ra2ce/analyses/analysis_config_data/analysis_config_data.py

```diff
@@ -36,18 +42,11 @@
 )
 
 IndirectAnalysisNameList: list[str] = [
-    "single_link_redundancy",
-    "multi_link_redundancy",
-    "optimal_route_origin_destination",
-    "multi_link_origin_destination",
-    "optimal_route_origin_closest_destination",
-    "multi_link_origin_closest_destination",
-    "losses",
-    "single_link_losses",
-    "multi_link_losses",
-    "multi_link_isolated_locations",
+    _enum.config_value for _enum in AnalysisIndirectEnum.list_valid_options()
+]
```

Comment:
based on what you've implemented you can simply:
```python
list(map(str, AnalysisIndirectEnum.list_valid_options))
```
Of course, test this :)

Reply:
Nice suggestion!